dashboard/public/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>QAT Dashboard</title>
+
+  <link rel='stylesheet' type='text/css' href='https://fonts.googleapis.com/css?family=Roboto:400,700'>
+  <link rel="stylesheet" href="http://ajax.googleapis.com/ajax/libs/angular_material/1.0.0/angular-material.min.css">
+  <link rel="stylesheet" href="css/app.css">
+
+  <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
```

Comment:
@christianvuerings I have been trying not to load in standard libraries from npm, instead relying on the browser/CDN to get cached versions. This would speed up first load for the dashboard, but will have no impact on subsequent loads (since everything will be cached. ) Do you think I should shift to loading from npm modules?